pkg/context/context.go

```diff
@@ -173,13 +176,16 @@ func (ctx *throwContext) Throw(err error) {
 func NewBackendContext4Test(withChangefeedVars bool) Context {
```

Comment:
maybe move test to _test file instead

Reply:
we use the `NewBackendContext4Test` function in another package, but a public function in `_test` file can't be import